releasenotes/notes/remove_schedule_from_backend_run-6b3c771799db9e9a.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
```

Comment:
I don't think this is a feature lol. I'm pretty sure there's a "removal" section.

Reply:
I couldn't find a `removal` section. Changed to `deprecations`.